app/src/main/java/fr/free/nrw/commons/CommonsApplication.java

```diff
@@ -44,21 +44,32 @@
 )
 public class CommonsApplication extends DaggerApplication {
 
+    public static final String DEFAULT_EDIT_SUMMARY = "Uploaded using Android Commons app";
+    public static final String FEEDBACK_EMAIL = "commons-app-android@googlegroups.com";
+    public static final String LOGS_PRIVATE_EMAIL = "commons-app-android-private@googlegroups.com";
+    public static final String FEEDBACK_EMAIL_SUBJECT = "Commons Android App (%s) Feedback";
     @Inject SessionManager sessionManager;
     @Inject DBOpenHelper dbOpenHelper;
     @Inject @Named("default_preferences") SharedPreferences defaultPrefs;
     @Inject @Named("application_preferences") SharedPreferences applicationPrefs;
     @Inject @Named("prefs") SharedPreferences otherPrefs;
-
-    public static final String DEFAULT_EDIT_SUMMARY = "Uploaded using Android Commons app";
-
-    public static final String FEEDBACK_EMAIL = "commons-app-android@googlegroups.com";
-    public static final String LOGS_PRIVATE_EMAIL = "commons-app-android-private@googlegroups.com";
-    public static final String FEEDBACK_EMAIL_SUBJECT = "Commons Android App (%s) Feedback";
-
     private CommonsApplicationComponent component;
     private RefWatcher refWatcher;
 
+    /**
+     * Provides a way to get member refWatcher
+     *
+     * @param context Application context
+     * @return application member refWatcher
+     */
+    public static RefWatcher getRefWatcher(Context context) {
+        CommonsApplication application = (CommonsApplication) context.getApplicationContext();
+        return application.refWatcher;
+    }
```

Comment:
Any particular reason to have moved this method?

Reply:
Auto code clean up may have been the cause of it

Reply:
Could you please move it back? It was intentionally just below the refWatcher declaration.
You can use the page https://github.com/commons-app/apps-android-commons/pull/1025/files to see exactly what you changed :-) Thanks!